com.unity.shadergraph/Editor/Drawing/MaterialGraphEditWindow.cs

```diff
@@ -244,7 +244,8 @@ void Update()
                     graphObject.isDirty = false;
                 }
 
-                graphEditorView.HandleGraphChanges();
+                // Why is this called twice?
```

Comment:
What value does this comment provide to future readers of this code? Find the answer and leave a comment to explain, or find it's not necessary and remove the double call. =)